lib/omnisharp-atom/features/go-to-implementation.ts

```diff
@@ -1,46 +1,42 @@
-import OmniSharpServer = require('../../omni-sharp-server/omni-sharp-server')
 import Omni = require('../../omni-sharp-server/omni')
 import OmniSharpAtom = require('../omnisharp-atom')
 
 class GoToImplementation {
-  private disposable: { dispose: () => void; }
-  private navigateToWord: string;
-  private atomSharper: typeof OmniSharpAtom;
-
-  constructor(atomSharper: typeof OmniSharpAtom) {
-      this.atomSharper = atomSharper;
-  }
-
-  public goToImplementation() {
-    var ref;
-    if (OmniSharpServer.vm.isReady) {
-        this.navigateToWord = (ref = atom.workspace.getActiveTextEditor()) != null ? ref.getWordUnderCursor() : void 0;
-        return Omni.goToImplementation();
+    private disposable: { dispose: () => void; }
+    private atomSharper: typeof OmniSharpAtom;
+
+    constructor(atomSharper: typeof OmniSharpAtom) {
+        this.atomSharper = atomSharper;
+    }
+
+    public goToImplementation() {
+        var editor = atom.workspace.getActiveTextEditor();
+        if (editor) {
+            var req: any = Omni.makeRequest();
+            req.word = <any>editor.getWordUnderCursor();
```

Comment:
I see it now, yeah that's probably a bug, looking at it.  I'll fix it after dinner is done.  It's not attaching the word to the request if the symbol can't be found.